Source/Core/Core/PatchEngine.cpp

```diff
@@ -257,6 +262,15 @@ static void ApplyPatches(const std::vector<Patch>& patches)
   }
 }
 
+static void ApplyMemoryPatches(const std::vector<std::size_t>& memory_patch_indices)
```

Comment:
I forget if we're completely over to C++20, but if we are, perhaps make this a `std::span`

```suggestion
static void ApplyMemoryPatches(std::span<const std::size_t> memory_patch_indices)
```

Reply:
Good point.  I'll try and see if buildbot complains :)

Reply:
Did you want to change this, or...?